### ISSUE TITLE
composer-plugin-api version issue?

### ISSUE THREAD

**Issue by yajra** [franzliedke/studio#46]
Doing a composer commands prompts an error/warning:
```
The "franzl/studio" plugin requires composer-plugin-api 1.0.0, this *WILL* break in the future and it should be fixed ASAP (require ^1.0 for example).
```

I think you need to release a new stable tag since it's fixed on master branch?

**Comment by tomicakorac** [comment]
Please search before you ask. This question has already been answered here:

https://github.com/franzliedke/studio/issues/43

In brief, you'll want to install the master branch to resolve this.

**Comment by yajra** [comment]
I already did tried to install the master before asking the issue but you cannot install master since it's not stable enough?

***composer global require franz/studio:dev-master***

```
Changed current directory to /Users/arjay/.composer
The "franzl/studio" plugin requires composer-plugin-api 1.0.0, this *WILL* break in the future and it should be fixed ASAP (require ^1.0 for example).
./composer.json has been updated
The "franzl/studio" plugin requires composer-plugin-api 1.0.0, this *WILL* break in the future and it should be fixed ASAP (require ^1.0 for example).
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package franz/studio could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

Anyways, thanks for the issue link. Didn't see that one.

**Comment by tomicakorac** [comment]
As @franzl wrote in #43, the other changes in master are not yet stable for a new tag. As soon as the release is ready, we will have this problem gone.
